.github/ISSUE_TEMPLATE.md

```diff
@@ -30,6 +30,9 @@ What behavior did you expect to observe?
 **Execution command:**
 <!-- e.g. `./foo 42`. If an input file is required, include it as well. -->
 
+**Associated Future Test(s):**
+<!-- Are there any tests in Chapel's test system that demonstrate this issue?
+     e.g. test/path/to/foo.future #1234 -->
```

Comment:
I thought about that too, but feel like the important stuff should come first, like a good news article, so am reluctant to move the source code down.  In particular, as I've filed these, I usually find the compile and execution commands to be pretty tedious (the bugs I've been filing aren't particularly sensitive to those commands, so I find myself always saying 'chpl foo.chpl' and 'a.out' (when I even get to the execution stage)).